### DISCUSSION TITLE
[a11y] Make sure all Sp7's actions can be accomplished without a mouse

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5553] [Feature Requests]
E.g, deleting members from an agent is not currently possible as the "remove" button is not focusable (EDIT: fixed)
Also, make sure all interactive elements have proper :focus and :active states to give visual feedback for keyboard and mouse users

**Comment by CarolineDenis** [comment]
@maxpatiiuk do we want to be able to navigate in the tree with the keyboard or just the actions buttons need to be available for keyboard?

**Comment by maxpatiiuk** [comment]
Tree viewer can already be navigated using the arrow keys, can't it?
And the tab/shift+tab could be used for focusing a different component of the UI (search bar or action buttons)
